Pushpendu

pushpendupurkait opened the pull request.

pushpendupurkait committed:
Merge branch 'development' of https://github.com/gupta-himanshu/UnoKIP into pushpendu

pushpendupurkait committed:
All unnecessary files from repository.

.classpath and .project files are deleted.

pushpendupurkait committed:
issues #11,#12,#13 solved.

11 test aggregate scoverage
12 sub-project example created.
13 dependencies pulled out.

pushpendupurkait committed:
samples project added

a new sub-project is added.

pushpendupurkait committed:
created a sample project.

A sub-project sample is added and sample domain is defined.
DBConnection package is changed to dbconnection.

phalodi committed:
Merge branch 'pushpendu' of https://github.com/gupta-himanshu/UnoKIP into sandeep

Conflicts:
	dbs/.gitignore
	dbs/src/test/scala/com/knoldus/db/ConnectionTest.scala

phalodi committed:
Add implicit reader and writer for the case class

-add reader and writer for case class
-change in crud operations

phalodi committed:
refactor

pushpendupurkait committed:
sample project refactored

sample project function added.

pushpendupurkait committed:
Merge branch 'sandeep' of https://github.com/gupta-himanshu/UnoKIP into pushpendu

Conflicts:
	dbs/src/main/scala/com/knoldus/dbcrud/DBCrud.scala

phalodi committed:
Unit test case for all crud operations

pushpendupurkait committed:
dbs is used in Samples project.

dbs is used in Samples project, all CRUD operations done.

phalodi committed:
Delete git ignore and remove conflicts

phalodi committed:
git ignore deleted from samples

pushpendupurkait committed:
json to case class conversion Done.

Json data to case class conversion is done in dbs Sub-Project.

phalodi committed:
include json converter in DBCrud

pushpendupurkait committed:
case class Object converted to Json Data.

case class object is converted to Json Data as well in dbs project.

phalodi committed:
Merge branch 'pushpendu' of https://github.com/gupta-himanshu/UnoKIP into sandeep

Conflicts:
	project/Build.scala
	project/Dependencies.scala

phalodi committed:
Merge branch 'pushpendu' of https://github.com/gupta-himanshu/UnoKIP into sandeep

phalodi committed:
Refactor code of DBCrud

->refactor insert method take input of json and convert it and add into database
->changes in test case of fetch

phalodi committed:
remove gitignore

gupta-himanshu commented:
@pushpendupurkait Is it necessary to keep .gitignore file in every sub-project ? If it is not then please remove them.

gupta-himanshu commented:
@pushpendupurkait Try to keep the "libraryDependencies" of a project inside its settings.

gupta-himanshu commented:
@pushpendupurkait Please separate out test dependencies from other dependencies.

pushpendupurkait committed:
gitignore deleted from sub folders.

gitignore file is deleted from every sub project

phalodi committed:
Change the input of all the crud operation with case class
and refactor the update function

 -change input of all crud operations with case class
 -refactor the update function
 -refactor code of test and sample

phalodi committed:
make a upsert equal to true so it will add document if not found

pushpendupurkait committed:
Issue #15 solved.

libraryDependencies is put inside the setting.
test dependencies is separated out.

phalodi committed:
Refactoring the code

-change check of update function
-add objectId in case class

phalodi committed:
Seprate the retrieve function from crud operations

-seprate retrieve operation from crud operation
-seprate from test case
-seprate from sample

phalodi committed:
Seprate out the find doc from from the crud operations in sample

phalodi committed:
Refactoring the code

phalodi committed:
Refactor the code

 - user can pass any collection name
 - change the traits into classes
 - refactor code of functions

pushpendupurkait committed:
new sub project added.

new sub project streaming added and samples project has been saperated.

Merge branch 'sandeep' of https://github.com/gupta-himanshu/UnoKIP into pushpendu

Conflicts:
	dbs/.gitignore
	samples/.gitignore

phalodi committed:
refactor findDoc sample code

pushpendupurkait committed:
findDoc is converted to generic

findDoc is coverted to generic
Merge branch 'sandeep' of https://github.com/gupta-himanshu/UnoKIP into pushpendu

Conflicts:
	samples/src/main/scala/com/knoldus/sample/CRUDSample.scala

gupta-himanshu commented:
@pushpendupurkait The **samples** module is not separated from other modules.
